### PR TITLE
Replace Registry using ENV provided by OpenShift CI

### PR DESCRIPTION
**Pull request by khrm**

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -20,6 +21,19 @@ readonly OPENSHIFT_BUILD_NAMESPACE=${OPENSHIFT_BUILD_NAMESPACE:-tektoncd-build-$
...
 
+function replaceImagesWithCIBuilds() {
+  RELEASE_YAML=$1
+  REGISTRY_PREFIX=$2
+
+  # Substitute images using the images built by OpenShift CI
+  # Ref: https://github.com/openshift/release/pull/21940
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-triggers-controller|${TEKTONCD_TRIGGERS_CONTROLLER}|g" ${RELEASE_YAML}
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-triggers-eventlistenersink|${TEKTONCD_TRIGGERS_EVENTLISTENERSINK}|g" ${RELEASE_YAML}
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-pipeline-interceptors|${TEKTONCD_TRIGGERS_INTERCEPTORS}|g" ${RELEASE_YAML}
```
Comment (savitaashture) [inline openshift/e2e-tests-openshift.sh:32]: tektoncd-pipeline-interceptors :thinking:

Comment (khrm) [reply openshift/e2e-tests-openshift.sh]: Good catch. Fixed this.

File: openshift/e2e-tests-openshift.sh [hunk 2]
```diff
@@ -20,6 +21,19 @@ readonly OPENSHIFT_BUILD_NAMESPACE=${OPENSHIFT_BUILD_NAMESPACE:-tektoncd-build-$
 # test-git-volume: `"gitRepo": gitRepo volumes are not allowed to be used]'
 declare -ar SKIP_YAML_TEST=(test-git-volume)
 
+function replaceImagesWithCIBuilds() {
+  RELEASE_YAML=$1
+  REGISTRY_PREFIX=$2
+
+  # Substitute images using the images built by OpenShift CI
+  # Ref: https://github.com/openshift/release/pull/21940
+  sed -i -e "s|${REGISTRY_PREFIX}:tektoncd-triggers-controller|${TEKTONCD_TRIGGERS_CONTROLLER}|g" ${RELEASE_YAML}
```
Comment (savitaashture) [inline openshift/e2e-tests-openshift.sh:30]: `TEKTONCD_TRIGGERS_CONTROLLER` to get ENV we need to have similar changes https://github.com/openshift/release/pull/21829/files for triggers also

Comment (savitaashture) [reply openshift/e2e-tests-openshift.sh:30]: Ah i see its done as part of this PR https://github.com/openshift/release/pull/21940

**Comment by savitaashture** [comment]
/lgtm

Thank you :+1:

**Pull request merged**